config/game_options.txt

```diff
@@ -122,6 +122,12 @@ PROTECT_ROLES_FROM_ANTAGONIST
 ## If non-human species are barred from joining as a head of staff, including jobs flagged as allowed for non-humans, ie. Quartermaster.
 #ENFORCE_HUMAN_AUTHORITY_ON_EVERYONE
 
+## Prevents non-human species from having their pay cut.
+#NO_NONHUMAN_WAGE_GAP
```

Comment:
We should avoid inverse names, it's a bit of a shame configs force us to act this way (I want to add migrations at some point, but I digress)

Can you rename it to something positive, like `EQUAL_NONHUMAN_WAGES`